### ISSUE TITLE
Setting gradient accumulation

### ISSUE THREAD

**Issue by d-ataman** [pytorch/fairseq#2380]
Hi,

I looked through the code and options but couldn't figure out how to set the number of steps to accumulate the gradients. Could you please direct me to some info or share the correct training argument?
Thanks a lot!
